lti_consumer/templates/email/user_score_notification.html

```diff
@@ -95,7 +95,6 @@ <h1>${_("Review available")}</h1>
                             </p>
                             <p style="color: rgba(0,0,0,.75);">
                                 Your Assignment: ${xblock_display_name} in course ${course_name} is reviewed.
```

Comment:
From vikas msg `when we get the email, it shows “courses”. It should say “OpenCV Courses”` should we also update this request in the email?